ens/main.py

```diff
@@ -210,7 +255,9 @@ def owner(self, name):
         return self.ens.caller.owner(node)
 
     @dict_copy
-    def setup_owner(self, name, new_owner=default, transact={}):
+    def setup_owner(
+        self, name: str, new_owner: Address=cast(Address, default), transact: TxDict=None
```

Comment:
We need a better pattern for this where we need to differentiate between `None` and not-provided..... Annoying that it ends up messing up the typing for things.